Add sst-2 ipynb example (#86)
bigscience-workshop/petals: README.md

print("Gradients (norm):", model.transformer.word_embeddings.weight.grad.norm())
```

> Of course, this is a simplified code snippet. For actual training, see our example on "deep" prompt-tuning here.

Reviewer: ```suggestion
Of course, this is a simplified code snippet. For actual training, see the example notebooks with "deep" prompt-tuning:
```

Author: Thank you! I'll make these changes.
